mu/locale/ru_RU/LC_MESSAGES/mu.po

```diff
@@ -0,0 +1,1618 @@
+msgstr ""
+"Не удалось получить список файлов на устройстве. Пожалуйста, проверьте логи Mu "
+"для получения технической информации. Или попробуйте отключить / подключить "
+"ваше устройство и / или перезапустить Mu."
+
+#: mu/interface/panes.py:767
+msgid ""
+"There was a problem copying the file '{}' onto the device. Please check Mu's "
+"logs for more information."
+msgstr ""
+"При копировании файла '{}' на устройство возникла проблема. Пожалуйста, "
+"проверьте логи Mu для получения дополнительной информации."
+
+#: mu/interface/panes.py:779
+msgid ""
+"There was a problem deleting '{}' from the device. Please check Mu's logs for "
+"more information."
+msgstr ""
+"При удалении \"{}\" с устройства возникла проблема. Пожалуйста, проверьте логи "
+"Mu для получения дополнительной информации."
+
+#: mu/interface/panes.py:791
+msgid ""
+"There was a problem getting '{}' from the device. Please check Mu's logs for "
+"more information."
+msgstr ""
+"Не удалось получить \"{}\" с устройства. Пожалуйста, проверьте логи Mu для "
+"получения дополнительной информации."
+
+#: mu/interface/widgets.py:70 mu/interface/widgets.py:141
+#: mu/interface/widgets.py:154
+msgid "No device connected."
+msgstr "Устройство не подключено."
+
+#: mu/logic.py:90
+msgid "Hello, World!"
+msgstr "Привет, Мир!"
+
+#: mu/logic.py:91
+msgid ""
+"This editor is free software written in Python. You can modify it, add features "
+"or fix bugs if you like."
+msgstr ""
+"Этот редактор представляет собой бесплатное программное обеспечение, написанное "
+"на Python. Вы можете изменить его, добавить функции или исправить ошибки, если "
+"хотите."
+
+#: mu/logic.py:95
+msgid "This editor is called Mu (you say it 'mew' or 'moo')."
+msgstr "Этот редактор называется Mu (произносится как «мяу» или «му»)."
+
+#: mu/logic.py:96
+msgid "Google, Facebook, NASA, Pixar, Disney and many more use Python."
+msgstr ""
+"Google, Facebook, NASA, Pixar, Disney и многие другие компании используют "
+"Python."
+
+#: mu/logic.py:97
+msgid ""
+"Programming is a form of magic. Learn to cast the right spells with code and "
+"you'll be a wizard."
+msgstr ""
+"Программирование - это магия. Научитесь произносить правильные заклинания с "
+"помощью кода, и вы станете волшебником."
+
+#: mu/logic.py:101
+msgid ""
+"REPL stands for read, evaluate, print, loop. It's a fun way to talk to your "
+"computer! :-)"
+msgstr ""
+"REPL означает «цикл „чтение — вычисление — вывод“». Это интересный способ "
+"поговорить с компьютером! :-)"
+
+#: mu/logic.py:105
+msgid "Be brave, break things, learn and have fun!"
+msgstr "Будьте смелыми, ломайте вещи, учитесь и получайте удовольствие!"
+
+#: mu/logic.py:106
+msgid "Make your software both useful AND fun. Empower your users."
+msgstr ""
+"Сделайте ваше программное обеспечение полезным и увлекательным. Расширьте "
+"возможности пользователей."
+
+#: mu/logic.py:107
+msgid "For the Zen of Python: import this"
+msgstr "Для Zen of Python: импортируйте это"
```

Comment:
`import this` is a command to type into the REPL.

Reply:
Although "Zen of Python" could be translated. An alternative English phrase might be, "the way of Python". Is this helpful..?